packages/formation-react/src/components/ErrorableCheckbox/ErrorableCheckbox.stories.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ErrorableCheckbox from './ErrorableCheckbox';
+
+export default {
+  title: 'Library/ErrorableCheckbox',
+  component: ErrorableCheckbox,
+  argTypes: {
+    required: { control: 'boolean' },
+    errorMessage: { control: 'text' },
+    id: { control: 'text' },
+    title: { control: 'text' },
+    label: { control: 'text' },
+    labelAboveCheckbox: { control: 'text' },
+  },
```

Comment:
I think the only ones you'll need here are `label` and `ariaLabelledBy` since those are functions in the props. The rest should be gotten automagically from the `propTypes` on the component.

Reply:
Actually I was able to remove the `argTypes` from this one as well since `defaultArgs` is passing the default content vals and the rest of the props can be inferred